.github/workflows/build_jupyterbook.yml

```diff
@@ -31,10 +31,11 @@ jobs:
         python -m pip install -r example_requirements.txt
         python -m pip install jupyter-book
 
-    # Build the book ensuring that build fails for warnings (-W flag)
+    # Build the book ensuring that the build treats warnings as errors (-W flag)
+    # CLI Reference: https://jupyterbook.org/en/stable/reference/cli.html
     - name: Build the book
       run: |
         cd examples
-        jupyter-book build -W .
+        jupyter-book build -W --verbose .
```

Comment:
Did you get any details about the fails?

Reply:
This doesn't work actually. Still need to figure out how to propogate the errors so that they can be viewed in CI.

I can remove this line for now.

Reply:
I asked about how to do this here
https://github.com/orgs/executablebooks/discussions/1086